jdaviz/configs/cubeviz/plugins/tests/test_parsers.py

```diff
@@ -239,11 +247,14 @@ def test_manga_with_mask(cubeviz_helper):
     uc.spectral_y_type = "Surface Brightness"
 
     se = cubeviz_helper.plugins['Spectral Extraction']
-    se.function = "Mean"
+    se.function = function
     se.extract()
-    extracted_max = cubeviz_helper.get_data("Spectrum (mean)").max()
-    assert_allclose(extracted_max.value, 5.566169e-18)
-    assert extracted_max.unit == u.Unit("erg / Angstrom s cm**2 pix**2")
+    extracted_max = cubeviz_helper.get_data(f"Spectrum ({function.lower()})").max()
+    assert_allclose(extracted_max.value, expected_value, rtol=5e-7)
+    if function == "Sum":
+        assert extracted_max.unit == u.Unit("erg / Angstrom s cm**2")
+    else:
+        assert extracted_max.unit == u.Unit("erg / Angstrom s cm**2 pix**2")
```

Comment:
Double checking – do we expect this unit difference?

Reply:
Yes, I think so. Sum over pixels remove the pix^2 unit.

Reply:
Yep, what @pllim said.